tests/models/test_identifier.py

```diff
@@ -660,41 +765,46 @@ def get_entry_dict(entry):
         assert 2 == len(entry.links)
-        assert any(filter(lambda l: l.href=='http://thumb', entry.links))
+        assert any(filter(lambda l: l.href == 'http://thumb', entry.links))
         # And the updated time has been changed accordingly.
-        expected = thumbnail.resource.representation.mirrored_at
         assert AtomFeed._strftime(even_later) == entry.updated
 
-    @parameterized.expand([
-        ('ascii_type_ascii_identifier_no_title', 'a', 'a', None),
-        ('ascii_type_non_ascii_identifier_no_title', 'a', 'ą', None),
-        ('non_ascii_type_ascii_identifier_no_title', 'ą', 'a', None),
-        ('non_ascii_type_non_ascii_identifier_no_title', 'ą', 'ą', None),
-
-        ('ascii_type_ascii_identifier_ascii_title', 'a', 'a', 'a'),
-        ('ascii_type_non_ascii_identifier_ascii_title', 'a', 'ą', 'a'),
-        ('non_ascii_type_ascii_identifier_ascii_title', 'ą', 'a', 'a'),
-        ('non_ascii_type_non_ascii_identifier_ascii_title', 'ą', 'ą', 'a'),
-
-        ('ascii_type_ascii_identifier_non_ascii_title', 'a', 'a', 'ą'),
-        ('ascii_type_non_ascii_identifier_non_ascii_title', 'a', 'ą', 'ą'),
-        ('non_ascii_type_ascii_identifier_non_ascii_title', 'ą', 'a', 'ą'),
-        ('non_ascii_type_non_ascii_identifier_non_ascii_title', 'ą', 'ą', 'ą'),
-    ])
-    def test_repr(self, _, identifier_type, identifier, title):
-        """Test that Identifier.__repr__ correctly works with both ASCII and non-ASCII symbols.
-
-        :param _: Name of the test case
-        :type _: str
-
-        :param identifier_type: Type of the identifier
-        :type identifier_type: str
-
-        :param identifier: Identifier's value
-        :type identifier: str
-
-        :param title: Presentation edition's title
-        :type title: str
+    @pytest.mark.parametrize(
+        'identifier_type, identifier, title',
+        [
+            ('a', 'a', None),
+            ('a', 'ą', None),
+            ('ą', 'a', None),
+            ('ą', 'ą', None),
+            ('a', 'a', 'a'),
+            ('a', 'ą', 'a'),
+            ('ą', 'a', 'a'),
+            ('ą', 'ą', 'a'),
+            ('a', 'a', 'ą'),
+            ('a', 'ą', 'ą'),
+            ('ą', 'a', 'ą'),
+            ('ą', 'ą', 'ą'),
+        ],
+        ids=[
+            'ascii_type_ascii_identifier_no_title',
+            'ascii_type_non_ascii_identifier_no_title',
+            'non_ascii_type_ascii_identifier_no_title',
+            'non_ascii_type_non_ascii_identifier_no_title',
+            'ascii_type_ascii_identifier_ascii_title',
+            'ascii_type_non_ascii_identifier_ascii_title',
+            'non_ascii_type_ascii_identifier_ascii_title',
+            'non_ascii_type_non_ascii_identifier_ascii_title',
+            'ascii_type_ascii_identifier_non_ascii_title',
+            'ascii_type_non_ascii_identifier_non_ascii_title',
+            'non_ascii_type_ascii_identifier_non_ascii_title',
+            'non_ascii_type_non_ascii_identifier_non_ascii_title'
```

Comment:
This is purely aesthetic, but it's more difficult for me to keep track of the parameters when the IDs are kept separately from the inputs. Can you confirm this setup makes more sense for you?